src/bin/vip-import-sql.js

```diff
@@ -265,6 +265,15 @@ If you are confident the file does not contain unsupported statements, you can r
 	return getTableNames();
 }
 
+function validateFilename( filename ) {
+	const re = /^[a-z0-9\-_\.]+$/i;
```

Comment:
```suggestion
	const re = /^[a-z0-9\-\_\.]+$/i;
```

nit pick, doesn't really matter but just to keep it uniform